### PR TITLE
Case 21025 conditionalizing TabletWebView features

### PR DESCRIPTION
**Pull request by roxanneskelly**
https://highfidelity.manuscript.com/f/cases/21025/marketplace-api-followup-conditionalizing-use-of-TabletWebView-feature

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18211/HighFidelity-Beta-PR15204-9182db8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18211/HighFidelity-Quest-Beta-PR15204-9182db8.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18211/HighFidelity-Beta-PR15204-9182db8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18211/HighFidelity-Beta-Interface-PR15204-9182db8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18211/HighFidelity-Beta-PR15204-9182db8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18211/HighFidelity-Beta-Interface-PR15204-9182db8.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18227/HighFidelity-Beta-PR15204-fe28eac.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18227/HighFidelity-Beta-Interface-PR15204-fe28eac.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18227/HighFidelity-Beta-PR15204-fe28eac.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18227/HighFidelity-Quest-Beta-PR15204-fe28eac.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18227/HighFidelity-Beta-PR15204-fe28eac.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18227/HighFidelity-Beta-Interface-PR15204-fe28eac.exe

File: interface/resources/qml/hifi/NameCard.qml [hunk 1]
```diff
@@ -376,8 +382,7 @@ Item {
     }
     FiraSansRegular {
         id: nameCardConnectionInfoText
-        visible: selected && !isMyCard && pal.activeTab == "connectionsTab"
-        width: parent.width
+        visible: selected && !isMyCard && pal.activeTab == "connectionsTab" && PlatformInfo.has3DHTML()
```
[the remark this answers is not in the document]
Comment (roxanneskelly) [reply interface/resources/qml/hifi/NameCard.qml]: I wanted to make sure has3DHTML was called only once for perf reasons, at least within the NameCard

File: interface/resources/qml/hifi/NameCard.qml [hunk 2]
```diff
@@ -364,8 +368,10 @@ Item {
         enabled: selected
         hoverEnabled: true
         onClicked: {
-            userInfoViewer.url = Account.metaverseServerURL + "/users/" + userName;
-            userInfoViewer.visible = true;
+            if(has3DHTML) {
```
[the remark this answers is not in the document]
Comment (roxanneskelly) [reply interface/resources/qml/hifi/NameCard.qml]: fixed

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18263/HighFidelity-Beta-PR15204-ab61f65.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18263/HighFidelity-Beta-Interface-PR15204-ab61f65.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18263/HighFidelity-Beta-PR15204-ab61f65.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18263/HighFidelity-Quest-Beta-PR15204-ab61f65.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18263/HighFidelity-Beta-PR15204-ab61f65.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18263/HighFidelity-Beta-Interface-PR15204-ab61f65.exe

**Pull request merged**